cmake/templates/BoostConfig.cmake.in

```diff
@@ -39,6 +39,18 @@ if(NOT TARGET "Boost::boost")
         INTERFACE_COMPILE_DEFINITIONS "BOOST_ALL_NO_LIB=1"
     )
   endif()
+
+  if(BOOST_CONFIG_LINK_OPENSSL)
+    hunter_add_package(OpenSSL)
```

Comment:
Also see note about OpenSSL imported targets: https://github.com/hunter-packages/websocketpp/pull/1#issuecomment-213371486

This one is important and should be solved first.